src/routes/safe/components/Settings/Advanced/index.tsx

```diff
@@ -42,7 +42,7 @@ const Advanced = (): React.ReactElement => {
   const classes = useStyles()
   const nonce = useSelector(safeNonceSelector)
   const modules = useSelector(safeModulesSelector)
-  const moduleData = getModuleData(modules) ?? null
+  const moduleData = modules ? getModuleData(modules) ?? null : null
```

Comment:
So getModuleData can return undefined?

Do we want to control this inside the function?

Reply:
I was just wondering if we should make the function to directly return `null` when result is undefined. Is not a big deal because it's only used here.
I feel that the logic to call the function gets a bit more complex to read just to ensure the result is null

Reply:
As I said I didn’t dive deep into the logic, my goal was to enable the option and make project compile

Reply:
As this method is only used here yet I would add the logic inside. Afterwards could be more complicated. But I leave the decision in your hands.